makefiles/Makefile.docs-k8s-operator

```diff
@@ -78,7 +78,13 @@ publish:
 
 next-gen-html:
 	# snooty parse and then build-front-end
-	@-echo ${SNOOTY_DB_PWD} | snooty build ${REPO_DIR} "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true" || exit 0;
+	@echo ${SNOOTY_DB_PWD} | snooty build ${REPO_DIR} "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true";
+	if [ $? -eq 1 ]; then \
+		exit 1; \
+	else \
+		exit 0; \
+	fi
+	#rsync -az --exclude '.git' ${REPO_DIR}/../../snooty ${REPO_DIR};
```

Comment:
I notice that some of these Makefiles have rsync for the copy and some of them use the cp. Some of them appear to have both. Can we standardize?